src/utils/runtime/src/lib.rs

```diff
@@ -173,7 +174,7 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
 
         // Enable DEBUG level for all other crates
         #[cfg(debug_assertions)]
-        let filter = filter.with_default(Level::DEBUG);
+        let filter = filter.with_default(Level::INFO);
```

Comment:
Please revert this line. We do want to enable DEBUG log by default for targets not sepecified above.